lib/src/migrators/module.dart

```diff
@@ -667,6 +685,14 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
     } else {
       addPatch(patchDelete(node.span));
     }
+    if (_state.useAllowed) {
+      _state.beforeFirstImport ??= node.span.start;
+      var afterSemicolon =
+          extendForward(extendThroughWhitespace(node.span), ';')?.end;
+      _state.afterLastImport = currentUrl.path.endsWith('.sass')
+          ? node.span.end
+          : afterSemicolon ?? node.span.end;
```

Comment:
It's a little confusing that `afterSemicolon` is computed even when it's unnecessary. I'd probably just make this an `if` statement to avoid that.

Reply:
Done